### PR TITLE
Rebrand Core/Zaragoza to Aragon OSx

### PR DESCRIPTION
**Pull request by Michael-A-Heuer**
## Description

Task: [APP-1080](https://aragonassociation.atlassian.net/browse/APP-1080)

**Note:** 
This task also updates the paths for the documentation workflows. 
The builders-portal was prepared for this change by https://github.com/aragon/builders-portal/pull/43

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [x] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[APP-1080]: https://aragonassociation.atlassian.net/browse/APP-1080?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/contracts-ethers/CHANGELOG.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
-# Aragon Core Contracts-ethers
+# Aragon OSx Contracts-ethers
```
Comment (brickpop) [inline packages/contracts-ethers/CHANGELOG.md:1]: Same

File: packages/contracts-ethers/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@aragon/core-contracts-ethers",
+  "name": "@aragon/osx-ethers",
   "version": "0.7.1-alpha",
-  "description": "Aragon V2 DAO Framework. Contract definition for Ethers.js",
+  "description": "The Aragon OSx contract definitions for Ethers.js",
```
Comment (brickpop) [inline packages/contracts-ethers/package.json]: Same

File: packages/contracts/CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Aragon Core Contracts
+# Aragon OSx Contracts
```
Comment (brickpop) [inline packages/contracts/CHANGELOG.md:1]: Same

File: packages/contracts/Releases.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Contract Releases
```
Comment (brickpop) [inline packages/contracts/Releases.md:1]: Is there an update on the script generating this file?

File: packages/contracts/docs/osx/01-how-it-works/01-core/02-permissions/index.md [hunk 5]
```diff
@@ -7,29 +7,29 @@ title: Permissions
 At Aragon, we believe that **DAOs are permission management systems**.
 Permissions between contracts and wallets allow a DAO to manage and govern its actions.
 
-Here, you will learn how the permissions in aragonOSx work, how they can be granted and revoked from wallets and contracts, and how they are managed through the DAO.
+Here, you will learn how the permissions in Aragon OSx work, how they can be granted and revoked from wallets and contracts, and how they are managed through the DAO.
 
 As we mentioned earlier, it is essential that only the right person or contract can execute a certain action. As a developer, you might have seen or used [modifiers such as `onlyOwner`](https://docs.openzeppelin.com/contracts/2.x/api/ownership#Ownable) in contracts. This `onlyOwner` modifier provides basic access control to your DAO: only the `owner` address is permitted to execute the function to which the modifier is attached.
 
 In aragonOSx, we follow the same approach but provide more advanced functionality:
```
Comment (brickpop) [inline packages/contracts/docs/osx/01-how-it-works/01-core/02-permissions/index.md:14]: Aragon OSx

File: packages/contracts/docs/osx/index.md [hunk 6]
```diff
@@ -4,9 +4,9 @@ title: aragonOSx
 
 ## The Contracts behind aragonOSx
```
Comment (brickpop) [inline packages/contracts/docs/osx/index.md:5]:
```suggestion
## The Contracts behind Aragon OSx
```

**Comment by brickpop** [comment]
General question, won't this collide with PR #265 ?

File: packages/contracts/docs/osx/index.md (hunk 6 above)
Comment (Michael-A-Heuer) [reply packages/contracts/docs/osx/index.md]: Fixed in 66151f518d01899579276fd6df43401e0a0e1d79

File: packages/contracts/docs/osx/01-how-it-works/01-core/02-permissions/index.md (hunk 5 above)
Comment (Michael-A-Heuer) [reply packages/contracts/docs/osx/01-how-it-works/01-core/02-permissions/index.md]: Fixed in 66151f518d01899579276fd6df43401e0a0e1d79

**Comment by Michael-A-Heuer** [comment]
> General question @Michael-A-Heuer, won't this collide with PR #265 ?

Yes, there will be some merge conflicts that must be resolved.

However, most of my changes should not affect Mathias's work and the conflicts will be easy to resolve

File: package.json [hunk 7]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "@aragon/core",
+  "name": "@aragon/osx",
```
Comment (brickpop) [inline package.json:2]: This will collide with the package.json under `packages/contracts`. I would remove it or use a different thing

**Review by brickpop — DISMISSED** [review]

File: package.json (hunk 7 above)
Comment (Michael-A-Heuer) [reply package.json:2]: @mathewmeconry didn't merge https://github.com/aragon/osx/pull/265 yet and can make these changes himself.

Comment (mathewmeconry) [reply package.json:2]: go ahead. I probably close #265 and start over

**Review by mathewmeconry — CHANGES_REQUESTED** [review]

File: packages/contracts/.gitignore [hunk 8]
```diff
@@ -5,7 +5,7 @@ node_modules
 coverage
 coverage.json
 typechain
-docs/generated
+docs/osx/03-reference-guide
```
[the remark this answers is not in the document]
Comment (Michael-A-Heuer) [reply packages/contracts/.gitignore:8]: Yes, here it is on `builders-portal/staging`: https://github.com/aragon/builders-portal/tree/staging/docs/osx

File: packages/contracts/Releases.md [hunk 9]
```diff
@@ -558,7 +562,7 @@ TokenFactory: 0x304eBcA6a98F3a2d4424388814ddbFf8904Bd1cE
 ---
 
 Time: 2022-03-15T17:52:52.904Z  
-Commit: [27ae34257d5274cb3daa64ff820f0b399c035ccb](https://github.com/aragon/zaragoza/commit/27ae34257d5274cb3daa64ff820f0b399c035ccb)  
+Commit: [27ae34257d5274cb3daa64ff820f0b399c035ccb](https://github.com/aragon/app/commit/27ae34257d5274cb3daa64ff820f0b399c035ccb)
```
[the remark this answers is not in the document]
Comment (Michael-A-Heuer) [reply packages/contracts/Releases.md]: Fixed in 424bda97342e97a6a35805e21eb8f63f0d33d69b

**Review by mathewmeconry — APPROVED** [review]

**Review by brickpop — APPROVED** [review]

**Pull request merged**